widgets/features/features.php

```diff
@@ -253,6 +253,7 @@ function get_less_variables( $instance ) {
 			}
 		}
 
+		$less_vars['responsive_breakpoint'] = siteorigin_panels_setting('mobile-width') . 'px';
```

Comment:
So, uh, how do I handle that? Do I just default to what it was previously or is there a setting I can pull from?